azurelinuxagent/ga/exthandlers.py

```diff
@@ -1099,20 +1102,27 @@ def collect_ext_status(self, ext):
         ext_status = ExtensionStatus(seq_no=seq_no)
         try:
             data_str = fileutil.read_file(ext_status_file)
+
+            if len(data_str) > MAX_STATUS_FILE_SIZE_IN_BYTES:
+                msg = "Handler - {0}, status file {1} of size {2} bytes is too big. Max Limit allowed is {3} " \
```

Comment:
if we are propagating those messages to users, they need to be more oriented to them. you may want to add a detailed message for the publisher (telemetry would also be good) and a friendlier message for the user